README.md

```diff
@@ -98,6 +141,53 @@ patient.patient_info.first_name
 
 **NOTE:** This is not intended to be used on all relationships! Only those where you intend to grant implicit access based on access to another model. In this use case, we assume that allowed access to `Patient` implies allowed access to `PatientInfo`, and therefore does not require an additional `allow_phi!` check. There are no guaranteed safeguards against circular `extend_phi_access` calls!
 
+### Check If PHI Access is allowed
+
+To check if PHI is allowed for a particular instance of a class call `phi_allowed?`.
+
+```ruby
+patient = Patient.new
+patient.phi_allowed? # => false
+
+patient.allow_phi('user@example.com', 'reason') do
+  patient.phi_allowed? # => true
+end
+
+patient.phi_allowed? # => false
+
+patient.allow_phi!('user@example.com', 'reason')
+patient.phi_allowed? # => true
+```
+
+This also works if access was granted at the class level
+
+```ruby
+patient = Patient.new
+patient.phi_allowed? # => false
+Patient.allow_phi!('user@example.com', 'reason')
+patient.phi_allowed? # => true
+```
+
+There is currently no class level equivalent for `phi_allowed?`.
```

Comment:
This would be stupidly easy to implement.

Reply:
Yes there is a task for it.